xmtp_mls/src/groups/message_history.rs

```diff
@@ -130,4 +210,13 @@ mod tests {
 
         let _ = amal_b.send_message_history_request().await;
     }
+
+    #[tokio::test]
+    async fn test_prepare_messages_to_send() {
+        let wallet = generate_local_wallet();
+        let amal_a = ClientBuilder::new_test_client(&wallet).await;
+        // let group_a = amal_a.create_group(None);
+        let messages_result = amal_a.prepare_messages_to_send().await;
```

Comment:
This test doesn't do a whole lot. Would be good to send a message and make sure that it gets added to the bundle

Reply:
The next PR will actually be the roundtrip-ing.. I'll for sure be adding a number of tests around that.  This test is really a stub for that, at this point.